.gitignore

```diff
@@ -10,3 +10,4 @@ yarn-error.log
 package-lock.json
 dist-*
 .vscode
+.lankrc.js
```

Comment:
@parkerziegler -- This is unrelated but useful so I can use https://github.com/FormidableLabs/lank#readme when working on both `inspectpack` and `webpack-dashboard` at the same time...

Reply:
Yep, I remember looking at `lank` for symlinking, all good here!